HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -4132,6 +3979,17 @@ def self.is_central_air_conditioner_and_furnace(hpxml, heating_system, cooling_s
     return true
   end
 
+  def self.is_ptac(hpxml, heating_system, cooling_system)
```

Comment:
Perhaps it would make more sense to rename the `is_central_air_conditioner_and_furnace` method to something generic (like `is_attached_heating_and_cooling_systems`) and just incorporate this logic into it?